docs/source/examples/formulas_examples.rst

```diff
@@ -0,0 +1,13 @@
+Formulas
```

Comment:
What would be the difference between text examples and formula examples?

Reply:
I like the idea of separating text and formulas, cause they are kind of different.
What I would include to them:
Text:
* How to set fonts
* How to scale, align, colour text
* How to use text to annotate things
* Show how text is used in different languages
* We could use examples from here https://github.com/3b1b/manim/pull/680 (and ask @xy-23 for permission)
 
Formulas:
* Show different use cases: integrals, derivatives, etc.
* Show how to label specific variables in certain colours and their limitations (e.g. I remember that it was not possible to colour variables that are written in a tex vector)
* Show how to align equations in a way that they look nicely (e.g. how to make equal signs with the same indent level)

Reply:
Ok this makes a lot of sense. One thing tho is that we don't want to write a whole LaTeX manual for the formulas part, so in that page we should probably link to a couple of other LaTex resources as well.

Reply:
> should probably link to a couple of other LaTex resources as well 
good point.